### ISSUE TITLE
Add a default visibility option for replies

### ISSUE THREAD

**Issue by edmael** [mastodon/mastodon#32717]
### Pitch

Right now toots and reply share the same "default" visibility setting.
I think it would be better if we can split that choice based on toot and reply: this way (for example) I can ensure - without having to check it every time - that my comments are unlisted and won't "contaminate" the timeline leaving space for the main toots of the instance.

### Motivation

Toots and replies are different kind of content and their visibility should be addressed differently allowing users to customize their preferred defaults.

**Comment by andreacorinti** [comment]
I agree: it would also help when creating a thread and replying to your own public post, forgetting to set the visibility on future replies and inadvertently flooding the local timeline
